great_expectations/dataset/pandas_dataset.py

```diff
@@ -184,6 +184,24 @@ def expect_column_to_exist(self, column,
                 "success": False
             }
 
+    @DocInherit
+    @DataSet.expectation(['column_list'])
+    def expect_columns_to_be(self, column_list,
+                               result_format=None, include_config=False, catch_exceptions=None, meta=None):
+        if len(list(self.columns)) != len(column_list):
```

Comment:
Why is this check necessary? Won't the equality check on line 196 catch the case of different lengths?

Reply:
Good catch. You're right.

That kind of check is necessary if `self.columns` or `column_list` inherit from `np.array`. That's how I wrote the code initially.

Since we have to convert everything to lists for JSON serialization, we might as well convert to lists early, which makes the check extraneous.

Thanks for the close review!